api/pkg/apis/projectcalico/v3/tier.go

```diff
@@ -38,8 +38,10 @@ type Tier struct {
 
 const (
 	AdminNetworkPolicyTierOrder         = float64(1_000)      // 1K
+	AdminTierOrder                      = float64(1_000)      // 1K
```

Comment:
```suggestion
	KubeAdminTierOrder                      = float64(1_000)      // 1K
```
For consistency with new names?